net/devif/devif_poll.c

```diff
@@ -925,6 +925,7 @@ int devif_poll(FAR struct net_driver_s *dev, devif_poll_callback_t callback)
 
       if (netdev_iob_prepare(dev, false, 0) != OK)
         {
+          nwarn("IOB Prepare failed for dev %s!", dev->d_ifname);
```

Comment:
```suggestion
          nwarn("IOB Prepare failed for dev %s!\n", dev->d_ifname);
```
Missing newline at the end.

Reply:
Done.